nomad/fsm.go

```diff
@@ -1798,7 +1798,7 @@ func (n *nomadFSM) restoreImpl(old io.ReadCloser, filter *FSMFilter) error {
 
 		// COMPAT(1.0): Allow 1.0-beta clusterers to gracefully handle
```

Comment:
We should probably adjust this comment (or even remove it, as the error message is pretty descriptive). Maybe add a `// DEPRECATED` to the 2 deprecated snapshot type constants above, as well?

Reply:
1d42abb

Reply:
or better yet: 8786619